src/backend/distributed/master/master_modify_multiple_shards.c

```diff
@@ -150,7 +150,9 @@ master_modify_multiple_shards(PG_FUNCTION_ARGS)
 
 	if (modifyQuery->commandType != CMD_UTILITY)
 	{
-		DeferredErrorMessage *error = ModifyQuerySupported(modifyQuery);
+		bool onlySimpleUpdates = true;
```

Comment:
I'd maybe just call this `multiShardUpdate` or `multiShardQuery` to make its meaning clear.